packages/wallet/src/validator/monitor-and-validate.ts

```diff
@@ -15,19 +15,20 @@ config({ path: resolve(__dirname, `../../../.env`) })
 
 /* Internal Imports */
 import * as RollupChain from './contracts/RollupChain.json'
-import { RollupStateValidator } from '../types'
+import { Address, RollupStateValidator } from '../types'
 import {
   DefaultRollupStateMachine,
   DefaultRollupStateValidator,
-  genesisState,
+  getGenesisState,
   RollupFraudGuard,
 } from '../index'
 
 const log = getLogger('monitor-and-validate')
 
-const rollupContractAddress = process.env.ROLLUP_CONTRACT_ADDRESS
-const mnemonic = process.env.WALLET_MNEMONIC
-const jsonRpcUrl = process.env.JSON_RPC_URL
+const aggregatorAddress: Address = process.env.AGGREGATOR_ADDRESS
+const rollupContractAddress: Address = process.env.ROLLUP_CONTRACT_ADDRESS
+const mnemonic: string = process.env.WALLET_MNEMONIC
```

Comment:
So I guess our mnemonic can potentially disagree with the `aggregatorAddress`? 

Maybe it should check that they are the same with something like: `Wallet.fromMnemonic(mnemonic).address === aggregatorAddress`?

Reply:
The mnemonic is the validator's mnemonic, not the aggregator's. So if the validator finds fraud, they'll submit a tx with their wallet.